pkg/kubectl/cmd/create_service.go

```diff
@@ -134,6 +134,7 @@ func NewCmdCreateServiceNodePort(f *cmdutil.Factory, cmdOut io.Writer) *cobra.Co
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.ServiceNodePortGeneratorV1Name)
+	cmd.Flags().Int32("node-port", 0, "Port used to expose the service on each node in a cluster.")
```

Comment:
Since [NodePort is of type `int32`](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L1931), I thought to receive it as that from the flag value, although I could receive it as `int` and convert it afterwards.